app/services/agent_removal.rb

```diff
@@ -25,5 +25,20 @@ def upcoming_rdvs?
   def should_soft_delete?
     (@agent.organisations - [@organisation]).empty?
   end
+
+  def errors
```

Comment:
On pourrait avoir un nom de méthode un peu plus descriptif ici
```suggestion
  def error_message
```